pyvista/core/dataobject.py

```diff
@@ -909,3 +909,7 @@ def _unserialize_pyvista_pickle_format(self: Self, state: dict[str, Any]) -> Non
         # copy data
         self.copy_structure(mesh)  # type: ignore[arg-type]
         self.copy_attributes(mesh)  # type: ignore[arg-type]
+
+    @property
+    def is_empty(self) -> bool:
+        """Return ``True`` if the object is empty."""
```

Comment:
Looks like mypy errors.  I'm assuming we cannot use ABC here for the class to use abstract method to force overwriting on the subclasses.  Should we `raise NotImplementedError` instead?  This _might_ make mypy happy.